content/exchange-guide.md

```diff
@@ -0,0 +1,575 @@
+
+1. [Deposit XRP into Exchange](#deposit-xrp-into-exchange)
+
+2. [Rebalance XRP Holdings](#rebalance-xrp-holdings)
+
+3. [Withdraw XRP from Exchange](#withdraw-xrp-from-exchange)
+
+4. [Trade XRP on the Exchange](#trade-xrp-on-the-exchange)
+
+
+This list does not include the [prerequisites](#prerequisites-for-supporting-xrp) required of an exchange.
+
+At this point, _Alpha Exchange_ has created [operational, standby, and issuing accounts](#accounts) on the RCL and added them to its balance sheet, but has not accepted any deposits from its users.
+
+
+<table>
+  <tr>
+    <td><b><i>XRP Balances
+on RCL</i></b></td>
+    <td></td>
+    <td></td>
+    <td><b><i>Alpha Exchange
+XRP Balances</i></b></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><b>User</b></td>
+    <td><b>Balance</b></td>
+    <td></td>
+    <td><b>Acct #</b></td>
+    <td><b>User</b></td>
+    <td><b>Balance</b></td>
+  </tr>
+  <tr>
+    <td>Dave</td>
+    <td>25,000</td>
+    <td></td>
+    <td>123</td>
+    <td>Alice</td>
+    <td>0</td>
+  </tr>
+  <tr>
+    <td>Edward</td>
+    <td>45,000</td>
+    <td></td>
+    <td>456</td>
+    <td>Bob</td>
+    <td>0</td>
+  </tr>
+  <tr>
+    <td>Charlie</td>
+    <td>50,000</td>
+    <td></td>
+    <td>789</td>
+    <td>Charlie</td>
+    <td>0</td>
+  </tr>
+  <tr>
+    <td><i>Alpha Operational</i></td>
+    <td>0</td>
+    <td></td>
+    <td>...</td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><i>Alpha Standby</i></td>
+    <td>0</td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><i>Alpha Issuing</i></td>
+    <td>0</td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>...</td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+</table>
+
+
+### Deposit XRP into Exchange
+
+To track [off-ledger XRP balances](#on-ledger-and-off-ledger) exchanges need to create new [balance sheets](#balance-sheets) (or similar accounting systems). The following table illustrates the balance changes that take place on Alpha Exchange's new balance sheet as users begin to deposit XRP.
+
+A user named Charlie wants to deposit 50,000 XRP to Alpha Exchange. Doing this involves the following steps:
+
+1. Charlie submits a payment of 50,000  XRP (by using [RippleAPI](https://ripple.com/build/rippleapi/) or similar software) to Alpha Exchange's [issuing account](#accounts).
+
+    a. Charlie adds an identifier (in this case, `789`) to the payment to associate it with his account at Alpha Exchange. This is called a [_destination tag_](https://ripple.com/build/gateway-guide/#source-and-destination-tags). (To use this, Alpha Exchange must have set the asfRequireDest flag on all of its accounts. This flag requires all incoming payments to have a destination tag like Charlie's. For more information, see [AccountSet Flags](https://ripple.com/build/transactions/#accountset-flags).
```

Comment:
> To use this, Alpha Exchange must have set the asfRequireDest flag on all of its accounts.

This is not strictly true. Having the RequireDest flag disabled provides no barriers to using destination tags at all; it just doesn't prevent users from omitting them.